experimental/packages/otlp-exporter-base/src/transport/fetch-transport.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { IExporterTransport } from '../exporter-transport';
+import { ExportResponse } from '../export-response';
+import { diag } from '@opentelemetry/api';
+import {
+  isExportRetryable,
+  parseRetryAfterToMills,
+} from '../is-export-retryable';
+
+export interface FetchTransportParameters {
+  url: string;
+  headers: () => Record<string, string>;
+}
+
+class FetchTransport implements IExporterTransport {
+  constructor(private _parameters: FetchTransportParameters) {}
+
+  async send(data: Uint8Array, timeoutMillis: number): Promise<ExportResponse> {
+    try {
+      const abortController = new AbortController();
+      const timeout = timeoutMillis
+        ? setTimeout(() => abortController.abort(), timeoutMillis)
+        : undefined;
+
+      const base = globalThis.location.href;
```

Comment:
This is browser specific. Are we 100% certain this will only ever run in browsers? Looks like right now it's only called from create legacy browser delegate so probably fine? I'd want @pichlermarc input on this since he's much more updated on the current code.

Reply:
thanks i think that's a good way to handle it